src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -1040,6 +1046,9 @@ protected boolean checkOrRequestItems(final boolean useItemDamage, @Nullable fin
         }
         itemsNeeded.clear();
         Collections.addAll(itemsNeeded, items);
+        //emergency workaround so the worker will tell what he needs (ignoring the content of his chest) since he is not able to check it in the issued state
+        final ItemStack itn = itemsCurrentlyNeeded.get(0);
```

Comment:
This is okayish for an emergency fix but definitly the wrong place to do this imo